### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by bshifaw**
- Updated WDLs to 1.0
- Removed GenerateOutputMap from bam2ubam because "this is not the best way to extract readgroup IDs. One issue is if other lines in the header contain the characters @RG, they will be caught here."
- Removed fc version of fastq2ubam because updates to Terra make working Data tables easier. Creating a seperate file with sample metadata isn't necessary.
- Added default variables to cram2bam

### CODE REVIEW

File: bam-to-unmapped-bams.wdl [hunk 1]
```diff
@@ -75,49 +62,27 @@ workflow BamToUnmappedBams {
...
+    Int disk_size
+    String docker
+    Int machine_mem_gb = 2
+    Int preemptible_attempts = 3
+  }
+    Int command_mem_gb = machine_mem_gb - 1    ####Needs to occur after machine_mem_gb is set 
 
-  command {
-    ${gatk_path} --java-options "-Xmx1000m" \
+  command {
```
Comment (kachulis) [inline bam-to-unmapped-bams.wdl:79]: I believe the `command <<<  >>>` with `~{ }` style is preferred over `command { }` with `${ }` as it frees up `$` from dual meanings of wdl placeholder and bash variable, which makes the code less confusing and bug prone.

File: bam-to-unmapped-bams.wdl [hunk 2]
```diff
@@ -75,49 +62,27 @@ workflow BamToUnmappedBams {
...
-  command {
-    ${gatk_path} --java-options "-Xmx1000m" \
+  command { 
+    PWD=`pwd`
+ 
+    ${gatk_path} --java-options "-Xmx${command_mem_gb}g" \
     RevertSam \
     --INPUT ${input_bam} \
-    --OUTPUT_MAP ${output_map} \
+    --OUTPUT $PWD \
```
Comment (kachulis) [inline bam-to-unmapped-bams.wdl]: You could just do `--OUTPUT .` instead

**Review by kachulis — COMMENTED** [review]
Only looked at the `bam-to-unmapped-bams.wdl` part, which looks good.

File: bam-to-unmapped-bams.wdl (hunk 2 above)
Comment (bshifaw) [reply bam-to-unmapped-bams.wdl]: done

File: bam-to-unmapped-bams.wdl (hunk 1 above)
Comment (bshifaw) [reply bam-to-unmapped-bams.wdl:79]: done

**Pull request merged**